examples/python/rippleadd-async.py

```diff
@@ -120,22 +120,36 @@ def unmajority(p, a, b, c):
 qobjs = []
 # Create online (so I/O bound) jobs if we have connetion or local (so CPU bound)
 # jobs otherwise
-if offline == False:
+if offline is False:
     print("Creating %d online jobs..." % NUM_JOBS)
     for _ in range(0, NUM_JOBS):
-        qobjs.append(qp.compile(["rippleadd"], backend=online_backend,
-                coupling_map=None, shots=1024))
+        qobjs.append(
```

Comment:
In general, would it be possible to try to favour keeping the amount of lines and structure similar to the existing code (bar the warnings, of course)? Ie. in this case (and in other instances on the PR), something along the lines of:
```python
        qobjs.append(qp.compile(["rippleadd"], backend=online_backend,
                                coupling_map=None, shots=1024))
```

It is quite debatable which form is more readable as a whole and there is a certain degree of subjectivity involved, though - so again, I see the point!

Reply:
Thanks for the feedback. I cringed a little for readability sake but I understand that formatting should be consistent throughout the entire app. I will make these changes when I get off work later :)